apps/detect/src/components/endpoint-overlay/endpoint-overlay.tsx

```diff
@@ -1,12 +1,39 @@
-import { Probe } from "@theprelude/core";
-import { Overlay, ProbeStatusIcon, ServerIcon } from "@theprelude/ds";
+import { useQuery } from "@tanstack/react-query";
+import {
+  Activity,
+  ActivityQuery,
+  formatDate,
+  Probe,
+  queries,
+  Test,
+} from "@theprelude/core";
+import {
+  AlertIcon,
+  CheckmarkIcon,
+  CloseIcon,
+  Overlay,
+  ProbeStatusIcon,
+  ServerIcon,
+} from "@theprelude/ds";
 import classNames from "classnames";
+import { addDays, subDays } from "date-fns";
 import styles from "./endpoint-overlay.module.css";
 
 const EndpointOverlay: React.FC<{
   endpoint: Probe;
   hideOverlay: () => void;
 }> = ({ endpoint, hideOverlay }) => {
+  const query: ActivityQuery = {
+    start: formatDate(subDays(new Date(), 7)),
+    finish: formatDate(addDays(new Date(), 1)),
+    endpoints: endpoint.endpoint_id,
+  };
+  const endpointInsights = useQuery({
+    ...queries.detect.activity.logs(query),
+    keepPreviousData: true,
+    suspense: false,
+  });
```

Comment:
```suggestion
  const filters = useFiltersStore(selectQuery)
  const endpointInsights = useQuery({
    ...queries.detect.activity.logs({...filters, endpoints: endpoint.endpoint_id}),
    keepPreviousData: true,
    suspense: false,
  });
```

This will apply the filters to the activity list